server_utils/origin.go

```diff
@@ -223,27 +231,103 @@ func GetOriginExports() (*[]OriginExport, error) {
 		}
 
 	case "s3":
-		// For now we're only supporting a single export for S3
-		// Our "federation prefix" is actually just
-		// /<Origin.S3ServiceName>/<Origin.S3Region>/<Origin.S3Bucket>
-		reads := (param.Origin_EnableReads.GetBool() || param.Origin_EnablePublicReads.GetBool())
-
-		federationPrefix := filepath.Join("/", param.Origin_S3ServiceName.GetString(),
-			param.Origin_S3Region.GetString(), param.Origin_S3Bucket.GetString())
-		originExport := OriginExport{
-			FederationPrefix: federationPrefix,
-			StoragePrefix:    "",
-			Capabilities: server_structs.Capabilities{
-				PublicReads: param.Origin_EnablePublicReads.GetBool(),
-				Writes:      param.Origin_EnableWrites.GetBool(),
-				Listings:    param.Origin_EnableListings.GetBool(),
-				Reads:       reads,
-				DirectReads: param.Origin_EnableDirectReads.GetBool(),
-			},
+		// Handle exports configured via -v or potentially env vars
+		if len(param.Origin_ExportVolumes.GetStringSlice()) > 0 {
+			log.Infoln("Configuring exports from export volumes passed via command line or via yaml")
+			for _, volume := range param.Origin_ExportVolumes.GetStringSlice() {
+				// Perform validation of the namespace
```

Comment:
Should there also be a check for the "/" at the beginning that we normally require for volume mounts